### PR TITLE
Feat/open accordion hash

### PR DESCRIPTION
**Pull request by Yanhuazong**
- Add id field to accordion block
- Open accordion if hashed
- Specify text color of Reason to believe block

### CODE REVIEW

File: src/accordion/block.js [hunk 1]
```diff
@@ -102,6 +103,14 @@ registerBlockType( 'purdue-blocks/accordion', {
               } }
             />
           </PanelRow>
+          <PanelRow>
+              <TextControl
+                label="HTML Anchor"
+                help="Enter a word without spaces to make a unique web address just for this block, called an “anchor.” Then, you’ll be able to link directly to this section of your page."
```
Comment (jyoung295) [inline src/accordion/block.js:109]: Probably specify more clearly here that the anchor needs to be unique from any other anchors on the page.

Comment (jyoung295) [inline src/accordion/block.js:109]: Probably specify more clearly here that the anchor needs to be unique from any other anchors on the page.

**Review by jyoung295 — CHANGES_REQUESTED** [review]

File: src/accordion/block.js (hunk 1 above)
Comment (Yanhuazong) [reply src/accordion/block.js]: Updated it to "Enter a word without spaces to make a unique web address just for this block, called an “anchor.” It must be unique from any other anchors on the page. Then, you’ll be able to link directly to this section of your page." Is it OK? I copied the description from the core blocks. Not sure if it's necessary to be this long.😅

Comment (jyoung295) [reply src/accordion/block.js]: That sounds good to me.

**Review by jyoung295 — APPROVED** [review]

**Pull request merged**